lib/Dialect/Comb/InferIntRangeInterfaceImpls.cpp

```diff
@@ -0,0 +1,256 @@
+//===- InferIntRangeInterfaceImpls.cpp - Integer range impls for comb -===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Implemenation of the interval range analysis interface.
+// The overflow flags are not set for the comb operations since they is
+// no meaningful concept of overflow detection in comb.
+//
+//===----------------------------------------------------------------------===//
+#include "circt/Dialect/Comb/CombOps.h"
```

Comment:
Super-nit: to be consistent with other files in CIRCT (also in the other files)
```suggestion
//===----------------------------------------------------------------------===//

#include "circt/Dialect/Comb/CombOps.h"
```